### PR TITLE
fix: LP fees reward handling + destination type

### LINKED ISSUE #9746: [Bug]: recurring transfer for LP fees received not distributed
### Problem encountered

In market-sim reward pools being funded with a dispatch metric set to `DISPATCH_METRIC_LP_FEES_RECEIVED` do not seem to be paying out to LPs.

https://jenkins.vega.rocks/job/common/job/vega-market-sim-reinforcement/56/artifact/fuzz_plots/rewards.jpg

Feature test added in #9745 to test behaviour.

In test we can see a recurring transfer is successfully setup but an LP who receives fees through a `TRANSFER_TYPE_LIQUIDITY_FEE_NET_DISTRIBUTE` transfer still receives no rewards.

Test checks both `entity_scope=ENTITY_SCOPE_INDIVIDUALS` and `entity_scope=ENTITY_SCOPE_TEAMS`.

### Observed behaviour

1. LP commits liquidity and submits pegged orders to meet their commitment
2. Market exits opening auction
3. Recurring transfer setup to fund an `ACCOUNT_TYPE_REWARD_LP_RECEIVED_FEES` pool
4. Trades generate fees
5. Move to end of epoch
6. Liquidity fees distributed
7. LP receives no rewards 🔴 

### Expected behaviour

1. LP commits liquidity and submits pegged orders to meet their commitment
2. Market exits opening auction
3. Recurring transfer setup to fund an `ACCOUNT_TYPE_REWARD_LP_RECEIVED_FEES` pool
4. Trades generate fees
5. Move to end of epoch
6. Liquidity fees distributed
7. LP receives full rewards 🟢 

### Steps to reproduce

```bash
1. cd vega
1. git checkout feat/rweard_lp_fees
1. cd core
1. go test -v ./integration/... --godog.format=pretty $(pwd)/integration/features/verified/rewards_lp_fees.feature
```

### Software version

08fb257b1aa456d355fe27aadbd8b4ca2a1f6cc8

### Failing test

integration/features/verified/rewards_lp_fees.feature

### Jenkins run

_No response_

### Configuration used

_No response_

### Relevant log output

_No response_

### LINKED ISSUE #9747: [Bug]: GQL API returning different transfer destination data than REST API
### Problem encountered

Governance initiated transfers - specifically tested on new transfers, but may also apply to cancel transfers - there is a difference between the reported destination value from GQL and REST.

### Observed behaviour

GQL:

Screenshot 2023-10-11 at 17 09 38

REST:

Screenshot 2023-10-11 at 17 09 59

Notice the "destination type" field has a different value. "ACCOUNT_TYPE_NETWORK_TREASURY" vs "ACCOUNT_TYPE_GENERAL"

### Expected behaviour

The two APIs to give the same correct value

### Steps to reproduce

```bash
Testable on a proposal on stagnet1.

On https://api.stagnet1.vega.rocks/graphql/ you can run this query:
{
  proposal(id: "dd9629cacfe528042bf5263c348546afdd941e433caa20eda5f820a454ff0e13") {
        terms {
          change {
            ...on NewTransfer {
              source
              sourceType
              destination
              destinationType
            }
          }
        }
      }
}

and the REST data can be found at https://api.n00.stagnet1.vega.xyz/api/v2/governance?proposalId=dd9629cacfe528042bf5263c348546afdd941e433caa20eda5f820a454ff0e13
```

### Software version

v0.73.0-preview.13

### Failing test

_No response_

### Jenkins run

_No response_

### Configuration used

_No response_

### Relevant log output

_No response_

### PR DESCRIPTION
**Pull request by ze97286**
Closes #9747 
close #9746

### CODE REVIEW

**Review by jeremyletang — DISMISSED** [review]

File: core/execution/common/market_activity_tracker.go [hunk 1]
```diff
@@ -434,7 +433,7 @@ func (mat *MarketActivityTracker) UpdateFeesFromTransfers(asset, market string,
 			mat.addFees(mt.makerFeesPaid, t.Owner, t.Amount.Amount, mt.totalMakerFeesPaid)
 		case types.TransferTypeMakerFeeReceive:
 			mat.addFees(mt.makerFeesReceived, t.Owner, t.Amount.Amount, mt.totalMakerFeesReceived)
-		case types.TransferTypeLiquidityFeeDistribute:
+		case types.TransferTypeLiquidityFeeNetDistribute:
 			mat.addFees(mt.lpFees, t.Owner, t.Amount.Amount, mt.totalLpFees)
```
Comment (cdummett) [inline core/execution/common/market_activity_tracker.go:437]: @ze97286 could we include `TransferTypeSlaPerformanceBonusDistribute` as well here?

**Review by cdummett — CHANGES_REQUESTED** [review]

File: core/execution/common/market_activity_tracker.go (hunk 1 above)
Comment (ze97286) [reply core/execution/common/market_activity_tracker.go:437]: I mean, we could, but should we?

Comment (cdummett) [reply core/execution/common/market_activity_tracker.go:437]: Also, if you are the only LP in a market and you only just meet the required time on book fraction, all your LP rewards will come to you in a `TransferTypeSlaPerformanceBonusDistribute` transfer and I would expect the LP to still be rewarded.

Comment (ze97286) [reply core/execution/common/market_activity_tracker.go]: I mean, not strictly, it's a metric for now much fees LP is getting, so why include bonuses? it's not like we're including penalties the other way around. Anyways, if you want it you got it.

Comment (cdummett) [reply core/execution/common/market_activity_tracker.go]: Yeah but the final fees the LP receives is NetDistribute + BonusDistribute.

Comment (cdummett) [reply core/execution/common/market_activity_tracker.go]: Nice thank you 🙏 

Market-sim plots are looking good now, will approve.

**Review by cdummett — DISMISSED** [review]

**Review by jeremyletang — APPROVED** [review]

**Pull request merged**